alerts/cluster-monitoring-operator/ThanosRuleEvaluationLatencyHigh.md

```diff
@@ -0,0 +1,49 @@
+# ThanosRuleEvaluationLatencyHigh
+
+## Meaning
+
+The alert fires when Thanos Ruler misses rule evaluations due to slow rule
+group processing.The alert is only about the user-defined recording and
+alerting rules.
+
+## Impact
+
+Alerts will take longer time to get delivered.
+
+## Diagnosis
+
+Check the logs for the Thanos Ruler pods:
+
+```console
+oc -n openshift-user-workload-monitoring logs -l 'thanos-ruler=user-workload'
+-c thanos-ruler
+```
```

Comment:
line 18 - 19 should be in the same line

Reply:
Thank you, I updated to have `\` at end of line, due to lint checks not possible to have more than 80 characters on one line